### PR TITLE
Feature/be select winner

### PR DESCRIPTION
**Pull request by KajenKirubah**
did back end for selecting a winner, just adds a winner field and sets to true on the chosen submission.

### CODE REVIEW

File: server/controllers/contest.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const Submission = require("../models/Submission");
+
+exports.getWinner = (req, res, next) => {
```
Comment (johnisr) [inline server/controllers/contest.js]: forgot async here, cannot use await without it and will crash

File: server/controllers/contest.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  
+  const subId = req.query.winner;
+  const { contestId } = req.params;
+
+  try {
+    await Contest.findByIdAndUpdate(contestId, {
```
Comment (johnisr) [inline server/controllers/contest.js:9]: is this setting the winner? should this be here in "getWinner" or is this supposed to be named setWinner?

Comment (johnisr) [reply server/controllers/contest.js:9]: also, remember to update status to "COMPLETED", could also be another check (other than not having winner property)

**Review by johnisr — APPROVED** [review]

**Comment by KajenKirubah** [comment]
the fe-selectWinner is more up to date

**Pull request closed without merging**